pkg/webhook/podhandler/podhandler.go

```diff
@@ -116,14 +122,9 @@ func (p *PodHandler) Handle(ctx context.Context, req admission.Request) admissio
 
 	marshaledPod, err := json.Marshal(pod)
 	if err != nil {
+		log.Error(errors.New("pod marshaling failed"), err.Error())
```

Comment:
this should also be reversed:
```suggestion
		log.Error(err, "pod marshaling failed")
```